input/cql/PCSBPScreeningFollowUpFHIR.cql

```diff
@@ -299,8 +312,11 @@ define "Encounter with Order for Hypertension Follow Up Declined by Patient":
 define "Second Hypertensive Reading SBP 130 to 139 OR DBP 80 to 89 Interventions Declined":
   ( ( ( [ServiceNotRequested: "Referral to Primary Care or Alternate Provider"]
         union [ServiceNotRequested: "Follow Up Within 6 Months"] ) SecondHTNDeclinedReferralAndFollowUp
-        where SecondHTNDeclinedReferralAndFollowUp.reasonRefused in "Patient Declined"
-          and SecondHTNDeclinedReferralAndFollowUp.status = 'completed'
+        where SecondHTNDeclinedReferralAndFollowUp.status = 'completed'
+          and exists(SecondHTNDeclinedReferralAndFollowUp.extension E 
+            where E.url = 'http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-doNotPerformReason' 
+            and E.value as Code in "Patient Declined"
+          )
```

Comment:
I think this is expected to work as `SecondHTNDeclinedReferralAndFollowUp.reasonRefused`

Reply:
Correct